cypress/integration/Packages/packages.ts

```diff
@@ -0,0 +1,181 @@
+/// <reference types="Cypress" />
+import {
+    ContentBuilder,
+    DocumentTypeBuilder
+  } from '../../../src';
+
+context('Packages', () => {
+    beforeEach(() => {
+        cy.umbracoLogin(Cypress.env('username'), Cypress.env('password'), false);
+      });
+
+      
+    function DeleteAllPackages(){
+        const url = 'https://localhost:44331/umbraco/backoffice/umbracoapi/package/GetCreatedPackages'
+        cy.umbracoApiRequest(url, 'GET', null).then((response) => {
+            response.forEach(function (value){
+                const requestUrl = 'https://localhost:44331/umbraco/backoffice/umbracoapi/package/DeleteCreatedPackage?packageId=' + value.id 
+                cy.umbracoApiRequest(requestUrl, 'POST', null)
+            });
+        });
+    }
+    function CreatePackage(contentId){
+        const newPackage = {
+            id: 0,
+            packageGuid: "00000000-0000-0000-0000-000000000000",
+            name: "TestPackage",
+            packagePath: "",
+            contentLoadChildNodes: false,
+            contentNodeId: contentId,
+            macros: [],
+            languages: [],
+            dictionaryItems: [],
+            templates: [],
+            partialViews: [],
+            documentTypes: [],
+            mediaTypes: [],
+            stylesheets: [],
+            scripts: [],
+            dataTypes: [],
+            mediaUdis: [],
+            mediaLoadChildNodes: false
+        }
+        const url = "https://localhost:44331/umbraco/backoffice/umbracoapi/package/PostSavePackage";
+        cy.umbracoApiRequest(url, 'POST', newPackage);
+    }
```

Comment:
Same as the other comment, here, I wonder if this should be moved, or even made into a builder pattern for better re-use? 

However, with Cypress and Umbraco limitations, I do realize that its use will quite limited, since we can't test if the content of the package is installed anyhow, so maybe this would be overkill and we should just keep this function as is for now?